webui/react/src/hooks/useCreateExperimentModal.module.scss

```diff
@@ -0,0 +1,42 @@
+.base:global(.ant-modal) {
```

Comment:
this seems like the most logical place to but css related to the hook... open to other locations